source/language/pulses.rst

```diff
@@ -108,10 +108,13 @@ Inline calibration blocks
 
 As calibration grammars may require the ability to insert top-level configuration information, perform program setup, or make inline calls
 to calibration-level instructions, OpenQASM supports the ability to declare a ``cal`` block. Within the ``cal`` block the
-semantics of the selected calibration grammar are valid. The ``cal`` block is of the same scope level as the enclosing block. The calibration
-grammar implementer may therefore choose to allow referencing or modifying values outside of the ``cal`` block but within the containing scope.
+semantics of the selected calibration grammar are valid. The calibration grammar implementer may therefore choose to
+allow referencing values (with its own syntax) from within the ``cal`` block that were declared within the containing parent scope.
+The ``cal`` block is of the same scope level as the enclosing block.
+The calibration grammar implementer may therefore choose to allow referencing or modifying values declared outside
+of the ``cal`` block but within the containing scope and its corresponding syntax.
```

Comment:
Looks like some text duplication here.

Reply:
Took another stab at the language in c1a6862.